README.md

```diff
@@ -123,7 +123,8 @@
 ### Node.js
 
 * [Napalm](https://github.com/nmattia/napalm) - Support for building npm packages in Nix with a lightweight npm registry.
-* [node2nix](https://github.com/svanderburg/node2nix) - Generate Nix expressions from a `package.lock` npm file.
+* [node2nix](https://github.com/svanderburg/node2nix) - Generate Nix expression from a `package-lock.json` (to be stored as files).
```

Comment:
node2nix does not use lock files.

Reply:
It certainly can use them https://github.com/svanderburg/node2nix#using-npm-lock-files. And I thought it internally generates the lock file but it does not seem to be the case. Opened https://github.com/nix-community/awesome-nix/pull/92.